integration/airflow/openlineage/airflow/adapter.py

```diff
@@ -277,22 +266,21 @@ def _build_run(
     ) -> Run:
         facets: Dict[str, BaseFacet] = {}
         if nominal_start_time:
-            facets.update({
-                "nominalTime": NominalTimeRunFacet(
-                    nominal_start_time, nominal_end_time
-                )
-            })
+            facets.update({"nominalTime": NominalTimeRunFacet(nominal_start_time, nominal_end_time)})
         parent_name = parent_job_name or job_name
         if parent_run_id is not None and parent_name is not None:
             parent_run_facet = ParentRunFacet.create(
                 runId=parent_run_id,
                 namespace=_DAG_NAMESPACE,
                 name=parent_name,
             )
-            facets.update({
-                "parent": parent_run_facet,
-                "parentRun": parent_run_facet,  # Keep sending this for the backward compatibility
-            })
+            facets.update(
+                {
```

Comment:
I hate this style :(
Can we change it?

Reply:
I didn't find any format options as such.
May I ask what's wrong with this style? :)

Reply:
Also: https://docs.astral.sh/ruff/formatter/black/

Reply:
Nothing, just that my eyes hurt. However, I believe pretty much _any_ sane enforced formatting is better than lack of it, so if we can't change it I'm okay with that.